easybuild/easyconfigs/m/Mesa/Mesa-19.1.7-GCCcore-8.3.0.eb

```diff
@@ -49,6 +49,7 @@ dependencies = [
     ('LLVM', '9.0.0'),
     ('X11', '20190717'),
     ('libunwind', '1.3.1'),
+    ('ncurses', '6.1'),
```

Comment:
I had to include `ncurses` as a dependency here to fix another failing part of the RPATH sanity check:

```
One or more required libraries not found for /software/Mesa/19.1.7-GCCcore-8.3.0/lib64/libGL.so.1.5.0:
...
libncurses.so.6 => not found
```

This is not required for more recent Mesa easyconfigs it seems, probably because `ncurses` is included there as an indirect dependency...

Reply:
OK